app/components/Form/ApplicationForm.tsx

```diff
@@ -204,7 +206,8 @@ const ApplicationForm: React.FC<Props> = ({
     mutationConfig?: Partial<
       UseDebouncedMutationConfig<updateApplicationMutation>
     >,
-    isRedirectingToNextPage = false
+    isRedirectingToNextPage = false,
+    isSaveAsDraftBtn = false
```

Comment:
When we have several optional parameters like this, its worth considering having a single `options` parameter instead. This will make the calling code easier to read.